.github/workflows/ci.yml

```
@@ -0,0 +1,110 @@
+        run: pnpm test:e2e
+        if: matrix.primary
+
+      - name: Upload test coverage
+        uses: codecov/codecov-action@v3
+        if: matrix.primary
+        with:
+          files: ./coverage/lcov.info
+          fail_ci_if_error: false
+
+      - name: Upload Playwright report
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)